week-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,43 @@
-function setAlarm() {}
+
+let variable;                                           // decleared three variable the purpose is shown inside "function startTime()".
+let timer;                   
+let countDownSeconds;
+
+function startTime() {
+  timer = document.getElementById("alarmSet");                 
+  countDownSeconds = parseInt(timer.value); 
```

Comment:
Great work on the code! In this function, you consider adding a validation check to prevent users from entering negative times. Or simply you can use the input for it:
```
<input id="alarmSet" type="number" min="0" /> // min="0"  will prevent the users entering negative numbers